packages/grafana-data/src/transformations/transformers/formatString.ts

```diff
@@ -0,0 +1,90 @@
+import { map } from 'rxjs/operators';
+
+import { Field, FieldType } from '../../types';
+import { DataTransformerInfo } from '../../types/transformations';
+
+import { DataTransformerID } from './ids';
+
+export enum FormatStringOutput {
+  UpperCase = 'Upper Case',
+  LowerCase = 'Lower Case',
+  FirstLetter = 'Capitalize First Letter',
+  EveryFirstLetter = 'Capitalize First Letter Of Every Word',
+  PascalCase = 'PascalCase',
+  CamelCase = 'camelCase',
+}
+
+export interface FormatStringTransformerOptions {
+  stringField: string;
```

Comment:
rather than using a raw string field, this should be switched to a fieldMatcher -- that will allow delegating some of the weirder complexities and support more options down the road

Reply:
also looks resolved